src/SDKs/TrafficManager/Management.TrafficManager/Microsoft.Azure.Management.TrafficManager.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.TrafficManager</PackageId>
     <Description>Microsoft Azure Management TrafficManager Library</Description>
     <AssemblyName>Microsoft.Azure.Management.TrafficManager</AssemblyName>
-    <VersionPrefix>2.3.0-preview</VersionPrefix>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
I'm increasing package and assembly major version due to the breaking changes and removing the -preview tag, given that we have stopped adding new features to the Hyak library and have switched investment to  Swagger. Is there any customer communication required?